modules/custom/az_news/az_news_feeds/az_news_feeds.services.yml

```
@@ -0,0 +1,7 @@
+services:
+  # Listens to migration events.
+  az_news_feeds.migrate_events_subscriber:
+    class: 'Drupal\az_news_feeds\EventSubscriber\AzNewsFeedsMigrateSubscriber'
+    arguments: ['@config.factory', '@plugin.manager.migration']
```

Doesn't look like the migration plugin manager service is actually used/needed (?)
```suggestion
    arguments: ['@config.factory']
```